src/app/loadout-drawer/loadout-drawer-reducer.ts

```diff
@@ -179,13 +179,6 @@ function addItem(
       dupe.amount += increment;
       // TODO: handle stack splits
     }
-
-    if (
-      draftLoadout.classType === DestinyClass.Unknown &&
-      item.classType !== DestinyClass.Unknown
-    ) {
-      draftLoadout.classType = item.classType;
-    }
```

Comment:
Note that I had to drop this because adding multiple subclasses to a loadout kept changing the class type when you didn't want to.